.github/ISSUE_TEMPLATE/issue.yml

```diff
@@ -0,0 +1,31 @@
+name: Issue
```

Comment:
This is the template for bugs, and there are even more templates 😅 You need to do each of them

Reply:
Can You elaborate what need to be added ? So I can add them

Reply:
You can take them from here, https://github.com/EddieHubCommunity/.github/tree/main/ISSUE_TEMPLATE